### DISCUSSION TITLE
A dumb question - what does this do?

### DISCUSSION THREAD

**Discussion by cheekoo** [fabianishere/udm-iptv#239] [General]
Does it enable multicast - as in one IPTV connection to watch on multiple devices? I ask this, because I use IPTV on my Android device without this configuration.
Thank you!

**Comment by fabianishere** [comment]
If you are already watching IPTV on your device, you probably don’t need this package.

This package is only necessary for ISPs that provide IPTV via IP multicast.

**Comment by cheekoo** [comment]
Thank you sir! Appreciate your answer and everything you do open-source!
